identity/handler.go

```diff
@@ -133,6 +136,54 @@ type getIdentityParameters struct {
 	ID string `json:"id"`
 }
 
+// swagger:model Credentials
+type CredentialRepresentation struct {
+	UpdatedAt time.Time `json:"UpdatedAt"`
+}
+
+// swagger:model Identity
```

Comment:
This will clash with the existing identity model - also, it should be lower case :)

```suggestion
// swagger:model identity
```

In order to not clash with the existing model, this would need to be changed to something else, e.g.


```suggestion
// swagger:model someOtherIdentity
```